web/frontend/src/components/slam.vue

```diff
@@ -177,49 +241,57 @@ const updateSLAM2dRefreshFrequency = async (name: string, time: 'manual' | 'off'
   }
 };
 
-const updateSLAM3dRefreshFrequency = async (name: string, time: 'manual' | 'off' | string) => {
-  refresh3DCancelled = true;
-  window.clearTimeout(slam3dTimeoutId);
-
-  if (time === 'manual') {
-    try {
-      const res = await fetchSLAMMap(name);
-      handleRefresh3dResponse(res);
-    } catch (error) {
-      handleError('fetchSLAMMap', error);
-    }
-  } else if (time === 'off') {
-    // do nothing
-  } else {
-    refresh3DCancelled = false;
-    scheduleRefresh3d(name, time);
-  }
-};
-
 const toggle2dExpand = () => {
   show2d = !show2d;
   updateSLAM2dRefreshFrequency(props.name, show2d ? selected2dValue : 'off');
 };
 
-const toggle3dExpand = () => {
-  show3d = !show3d;
-  updateSLAM3dRefreshFrequency(props.name, show3d ? selected3dValue : 'off');
+const selectSLAM2dRefreshFrequency = () => {
+  updateSLAM2dRefreshFrequency(props.name, selected2dValue);
 };
 
-const selectSLAM2dRefreshFrequency = () => {
+const refresh2dMap = () => {
   updateSLAM2dRefreshFrequency(props.name, selected2dValue);
 };
 
-const selectSLAMPCDRefreshFrequency = () => {
-  updateSLAM3dRefreshFrequency(props.name, selected3dValue);
+const handle2dRenderClick = (event: THREE.Vector3) => {
+  updatedDest = true;
+  threeJPos = event;
+  threeJPos.y = z - threeJPos.z;
 };
 
-const refresh2dMap = () => {
-  updateSLAM2dRefreshFrequency(props.name, selected2dValue);
+const handleUpdateX = (event: CustomEvent<{ value: string }>) => {
+  threeJPos.x = Number.parseFloat(event.detail.value);
+  updatedDest = true;
+};
+
+const handleUpdateY = (event: CustomEvent<{ value: string }>) => {
+  threeJPos.y = Number.parseFloat(event.detail.value);
+  updatedDest = true;
 };
 
-const refresh3dMap = () => {
-  updateSLAM3dRefreshFrequency(props.name, selected3dValue);
+const baseCopyPosition = () => {
+  const position = {
+    x,
+    y,
+    z,
+    o_x: oX,
+    o_y: oY,
+    o_z: oZ,
+    theta,
+  };
+
+  const asString = JSON.stringify(position);
+  copyToClipboardWithToast(asString);
```

Comment:
[q] Is this name a pattern in other parts of RDK? Specifically the use of the word clipboard

Reply:
Clipboard is correct here, specifically because we're calling the browser's Clipboard API